maintainers/maintainer-list.nix

```diff
@@ -4045,6 +4045,12 @@
     githubId = 12491746;
     name = "Masato Yonekawa";
   };
+  hzeller = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add hzeller`` and sort it alphabetically.

Reply:
Done: #119428

Reply:
Seperate commit, not seperate PR

Reply:
Ah, yes, that I did actually as a separate commit: https://github.com/NixOS/nixpkgs/pull/119394/commits/569c9a1b767a51c3c72cf0f44ea95759b574ae7a
.. but I didn't know the commit message naming conventions back then, so the commit message is different. I am not sure if I can amend the message from a commit that is not at head, or can I ?